dll/win32/feclient/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+spec2def(feclient.dll feclient.spec)
+
+list(APPEND SOURCE
+    ${CMAKE_CURRENT_BINARY_DIR}/feclient_stubs.c
+    ${CMAKE_CURRENT_BINARY_DIR}/feclient.def)
+
+add_library(feclient MODULE ${SOURCE})
+set_module_type(feclient win32dll UNICODE ENTRYPOINT 0)
+target_link_libraries(feclient wine)
+
+add_importlibs(feclient user32 kernel32 ntdll)
+add_cd_file(TARGET feclient DESTINATION reactos/system32 FOR all)
+set_wine_module(feclient)
```

Comment:
If I were you I would move this line up to just before or after you add the wine target link library.